services/github/github-go-mod.service.js

```diff
@@ -45,23 +45,14 @@ module.exports = class GithubGoModGoVersion extends ConditionalGithubAuthV3Servi
       keywords,
     },
     {
-      title: 'GitHub go.mod Go version (subfolder of monorepo)',
+      title: 'GitHub go.mod Go version (subdirectory of monorepo)',
       pattern: ':user/:repo',
       namedParams: { user: 'golang', repo: 'go' },
       queryParams: { filename: 'src/go.mod' },
       staticPreview: this.render({ version: '1.14' }),
       documentation,
       keywords,
     },
-    {
-      title: 'GitHub go.mod Go version (branch & subfolder of monorepo)',
-      pattern: ':user/:repo/:branch',
-      namedParams: { user: 'golang', repo: 'go', branch: 'master' },
-      queryParams: { filename: 'src/go.mod' },
-      staticPreview: this.render({ version: '1.14' }),
-      documentation,
-      keywords,
-    },
```

Comment:
It's not clear to me why we'd remove the example that allows users to specify a custom branch. Could you explain the rationale for removing this?

Reply:
Minimalizm :) I've contemplated that since the examples on specifying both `branch` and `filename` are already there, there is no need for this one. Reading your suggestions from other PR I realized someone might use this example to generate an URL. Hence I restored this example and added corresponding to the new badge.